lib/Bindings/Python/circt/design_entry.py

```diff
@@ -83,3 +83,11 @@ def body_build(mod):
           body_builder=body_build)
 
   return __Module
+
+
+def connect(destination, source):
+  builder = destination.builder
```

Comment:
Should probably throw some informative exception here if `destination` wasn't created the builder.

Reply:
(damn type hints not meaning anything a runtime.)